plasmapy/tests/helpers/tests/sample_functions.py

```diff
@@ -0,0 +1,173 @@
+"""
+Sample functions and classes to be used for testing the test helper
+functionality.
+"""
+
+import astropy.units as u
+import numpy as np
+import warnings
+
+from typing import NoReturn
+
+
+class SampleException(Exception):
+    """A sample exception to be used for testing purposes."""
+
+    pass
+
+
+class SampleExceptionSubclass(SampleException):
+    """
+    The subclass of the `SampleException` to used for testing purposes.
+    """
+
+    pass
+
+
+class SampleWarning(Warning):
+    """A sample warning to be used for testing purposes."""
+
+    pass
+
+
+class SampleWarningSubclass(SampleWarning):
+    """
+    The subclass of the `SampleWarning` to be used for testing purposes.
+    """
+
+    pass
+
+
+def return_42() -> int:
+    """A sample function for testing purposes that returns ``42``."""
+
+    return 42
+
+
+def return_42_meters() -> u.Quantity:
+    """
+    A sample function for testing purposes that returns an
+    `~astropy.units.Quantity` with a value of ``42.0 m``.
+    """
+
+    return 42.0 * u.m
+
+
+def return_np_array(*args) -> np.array:
```

Comment:
```suggestion
def return_np_array(args) -> np.array:
```

This allows you to do `return_np_array([1, 2, 4])` vs `return_np_array(1, 2, 3)`.  It also has a better match to the `np.array` signature.